modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -129,7 +129,7 @@ endif::upi[]
 `VirtualMachine.Provisioning.DeployTemplate`
 
 |vSphere vCenter Datacenter
-|If the installation program creates the virtual machine folder. For UPI, `VirtualMachine.Inventory.Create` and `VirtualMachine.Inventory.Delete` privileges are optional if your cluster does not use the Machine API.
+|If the installation program creates the virtual machine folder. For user-provisioned infrastructure, `VirtualMachine.Inventory.Create` and `VirtualMachine.Inventory.Delete` privileges are optional if your cluster does not use the Machine API. See the Minimum permissions for the Machine API table. 
```

Comment:
```suggestion
|If the installation program creates the virtual machine folder. For user-provisioned infrastructure, `VirtualMachine.Inventory.Create` and `VirtualMachine.Inventory.Delete` privileges are optional if your cluster does not use the Machine API. See the "Minimum permissions for the Machine API" table. 
```
Elsewhere in table, quotes are used in this reference for this table so I would be consistent.